Code.gs

```diff
@@ -292,12 +296,7 @@ function ConvertToCustomEvent(vevent){
   }
   else{
     event.startTime = icalEvent.startDate.toJSDate();
-    if (icalEvent.endDate == null){
```

Comment:
Obsolete as icalEvent.endDate will never be null. If the vevent is missing dtend ical.js will calculate an appropriate dtend according duration property (if it exists).

If duration is not specified
- dtend will be dtstart if Value=Date-Time
- dtend will be dtstart + 1d if Value=Date